docs/handlers/system-handlers.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Internal Route Handlers
+sidebar_label: Internal Route Handlers
+---
+
+The Zuplo Runtime automatically registers certain routes on your gateway to
+provide enhanced functionality. You may see requests to these routes displayed
+within your Analytics page. Below is a list of routes we have reserved:
+
+| Name                    | Method  | Path                              | Description                                                                                |
+| ----------------------- | ------- | --------------------------------- | ------------------------------------------------------------------------------------------ |
+| build-data              | GET     | `/__zuplo/build`                  | Provides information about the latest build of the gateway.                                |
```

Comment:
let's not share the `build-data` one for now or the path for this. It's not available to most folks I think (secured) so doesn't make sense. Let's just filter from the results?

Reply:
Sure FYI @vazexqi on filtering